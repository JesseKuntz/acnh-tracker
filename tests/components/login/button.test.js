import { h } from 'preact';
import LoginButton from '../../../src/components/login/button';

import { shallow } from 'enzyme';

describe('LoginButton', () => {
  test('matches snapshot', () => {
    expect(shallow(<LoginButton />)).toMatchSnapshot();
  });
});
