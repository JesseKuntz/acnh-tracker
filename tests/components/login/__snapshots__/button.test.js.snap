// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButton matches snapshot 1`] = `
<div
  className="container"
>
  <button
    className="login"
    onClick={[Function]}
  >
    Log In
  </button>
</div>
`;
